src/sqlfluff/dialects/dialect_mysql.py

```diff
@@ -62,7 +65,7 @@
         CodeSegment,
         name="quoted_literal",
         type="literal",
-        trim_chars=('"',),
+        trim_chars=('"'),
```

Comment:
It looks like here, you changed a tuple that contains a string to just be a string. Was this change intentional? If not, can you change it back?

Reply:
This has been fixed now